jaxrs-api/src/test/java/javax/ws/rs/core/testrules/DummyRuntimeDelegate.java

```diff
@@ -0,0 +1,41 @@
+package javax.ws.rs.core.testrules;
+
+import javax.ws.rs.core.*;
+import javax.ws.rs.ext.RuntimeDelegate;
+
+/**
+ * Dummy implementation of the RuntimeDelegate. This is needed to test classes that require a RuntimeDelegate.
+ * Currently all methods just return null values.
+ */
+public class DummyRuntimeDelegate extends RuntimeDelegate {
```

Comment:
Open discussion: Do we want to collect such dummies? Personally I love Mockito... ;-)

Reply:
I have created #604 for adding the mockito dependency. I guess a change like that warrants its own PR. I am not sure what has to be checked before adding a depedency (allowed licenses etc.).

Reply:
I also just noticed there is already a similar class javax.ws.rs.core.RuntimeDelegateStub that is used in other tests.

Reply:
Waiting for approval of the CQ I filed (https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15982).

Reply:
This may be an unpopular opinion, but I'm very conservative about adding new dependencies. In case of specifications like JAX-RS we should be very careful about dependencies, even for the `test` scope. JAX-RS should last for at least the next decade, and maintenance is usually simpler with fewer dependencies. Even if we have to create mocks our self.

However, as mentioned above, I'm very (maybe too) conservative about this. So if I'm the only one having doubts about add Mockito, just ignore my concern. :wink:

Reply:
I share your doubts in general, but Mockito in particular is way past its hype and has become a de-facto standard in lots of projects I contribute to. For me, JUnit-Hamcrest-Mockito meanwhile has became an indivisible default for Java projects.

Reply:
CQ passed.

Reply:
@mkarg I agree that Mockito is the most successful mocking library for Java out there. Therefore, I would be OK with adding it. However, I just wanted to note that we should take special care when adding new dependencies. Even for the test scope.